packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -247,16 +257,17 @@ export default class Menu extends Component {
         to: isAdmin ? 'xoa/update' : 'xoa/notifications',
         icon: 'menu-xoa',
         label: 'xoa',
-        extra: (
-          <span>
-            {isAdmin && (
-              <span>
-                <UpdateTag />{' '}
-              </span>
-            )}
-            <NotificationTag />
-          </span>
-        ),
+        extra:
+          noNotifications && (!isAdmin || xoaState === 'upToDate') ? null : (
+            <span>
+              {isAdmin && (
+                <span>
+                  <UpdateTag />{' '}
+                </span>
+              )}
+              <NotificationTag />
```

Comment:
It would be nice to also not have 2 icons at the same time when collapsed.

What about this?
- `extra` *can* be an array
- if `extra` is an array, we show all the extras when uncollapsed and only the first non-`null` one when collapsed

I think this is what requires the fewest changes because we can keep using the fact that `UpdateTag` and `NotificationTag` can render as `null`.